### ISSUE TITLE
508-defect-2 [FOCUS]: Focus should be set to the PDF download instruction modal's container, not the close button

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#33348]
#### VFS team name
Decision Tools

#### VFS product name
Find a VA Form PDF Download Instructions

#### Point of Contact/Reviewers
Josh Kim (@joshkimux) - Accessibility

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Focus is currently sent to the close button upon opening the modal. Despite this being the  default pattern for most modals, it's not ideal in this particular context where the modal appears only once upon initial interaction. Should a screen reader user accidentally close out of it suddenly, the message will be permanently lost. 

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
To prevent screen reader users from accidentally closing out the modal immediately, we recommend sending focus to the modal container itself. To ensure the user hears the full modal message upon activating the modal itself, we also recommend to use `aria-describedby` to point to its contents. For example:

`<div class="va-modal" role="dialog" aria-labelledby="va-modal-title" aria-describedby="va-modal-desc" aria-modal="true" tabindex="-1">`...
`<div id="va-modal-desc"><p>Download this PDF to your desktop computer or laptop. Then use Adobe Acrobat Reader to open and fill out the form. Don’t try to open the PDF on a mobile device or fill it out in your browser.</p>`...

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by shiragoodman** [comment]
@joshkimux can you please update the title of this ticket? thx!

**Comment by srsuddath** [comment]
@joshkimux this fix is live on staging if you wouldn't mind checking it out. We highlight the Modal title rather than the close button

**Comment by joshkimux** [comment]
<img width="554" alt="Screen Shot 2021-12-22 at 10 02 53 AM" src="https://user-images.githubusercontent.com/14154792/147113443-977c029b-5d7b-4333-87e5-59872271c7e7.png">

Confirmed! Great stuff 🥳
